Apps/W1/Shopify/app/src/Base/Tables/ShpfyShop.Table.al

```diff
@@ -507,33 +553,46 @@ table 30102 "Shpfy Shop"
         end;
     end;
 
-    internal procedure GetLastSyncTime(Type: Enum "Shpfy Synchronization Type"): DateTime
+    internal procedure GetLastSyncTime(Type: enum "Shpfy Synchronization Type"): DateTime
     var
-        SyncInfo: Record "Shpfy Synchronization Info";
+        ShpfySynchronizationInfo: Record "Shpfy Synchronization Info";
     begin
-        if SyncInfo.Get(Rec.Code, Type) then
-            exit(SyncInfo."Last Sync Time");
+        if Type = "Shpfy Synchronization Type"::Orders then begin
+            if Rec."Shop Id" = 0 then begin
+                CalcShopId();
+                Rec.Modify();
+            end;
+            if ShpfySynchronizationInfo.Get(Format(Rec."Shop Id"), Type) then
+                exit(ShpfySynchronizationInfo."Last Sync Time");
+        end;
+        if ShpfySynchronizationInfo.Get(Rec.Code, Type) then
+            exit(ShpfySynchronizationInfo."Last Sync Time");
         exit(0DT);
     end;
 
-    internal procedure SetLastSyncTime(Type: Enum "Shpfy Synchronization Type")
+    internal procedure SetLastSyncTime(Type: enum "Shpfy Synchronization Type")
     begin
         SetLastSyncTime(Type, CurrentDateTime);
     end;
 
-    internal procedure SetLastSyncTime(Type: Enum "Shpfy Synchronization Type"; ToDateTime: DateTime)
+    internal procedure SetLastSyncTime(Type: enum "Shpfy Synchronization Type"; ToDateTime: DateTime)
     var
-        SyncInfo: Record "Shpfy Synchronization Info";
+        ShpfySynchronizationInfo: Record "Shpfy Synchronization Info";
```

Comment:
There's inconsistent naming throughout the app. Sometimes the variable has the Shpfy prefix, sometimes it doesn't. Sometimes the prefix is Shopify.
It think it's fine to drop the Shpfy prefix. In most cases it doesn't add anything. In some cases, dropping the prefix makes the name a little confusing so there, adding Shopify seems the right thing to do.
ex: Record "Shpfy Customer" would be confusing as just Customer, but perfectly ok as ShopifyCustomer.

What is worse though is when the same object has different names in different places. For example, I have seen "Shpfy Shop" named as ShpfyShop, ShopifyShop and just Shop.

please make the naming consistent throughout the code.